ang/civicase/contact/directives/contact-icon.directive.html

```diff
@@ -9,6 +9,7 @@
     <span ng-switch="contactIcon">
       <i class="material-icons" ng-switch-when="Individual">person</i>
       <i class="material-icons" ng-switch-when="Household">home</i>
+      <i class="material-icons" ng-switch-when="Organization">domain</i>
```

Comment:
@deb1990 , Regarding the name change from `Organisation` to `Organization`, This probably never happened and most likely the code wasn't working properly before.
I looked through Civi code for previous versions and the name `Organization` seems to have been consistent. https://github.com/civicrm/civicrm-core/tree/4.3/xml/schema/Contact.
We should just remove the support for `Organisation`